client.go

```diff
@@ -2178,6 +2178,67 @@ func (c *ToznySDKV3) ShareSecretWithUsername(ctx context.Context, params ShareSe
 	return nil
 }
 
+type UnshareSecretInfo struct {
```

Comment:
I think elsewhere you use options, pick one of options / info / params and use consistently please.